Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -153,6 +171,19 @@ open class ChatMessageListVC:
         listView.backgroundColor = appearance.colorPalette.background
     }
 
+    override open func updateContent() {
+        super.updateContent()
+
+        listView.delegate = self
```

Comment:
nit: it's fine to have it here just curious if there was a strong reason to move the delegate assignment outside list view builder closure?

Reply:
There wasn't, it was to make the code more cleaner and tidy :)